strategies/affected/src/test/java/org/arquillian/smart/testing/strategies/affected/ClassDependenciesGraphTest.java

```diff
@@ -141,6 +145,48 @@ public void should_detect_all_changes_transitive() {
     }
 
     @Test
+    public void should_detect_all_changes_adding_package_annotated_transitive() {
```

Comment:
Can we have an ftest covering this new functionality?

Reply:
Yes, but not done yet until all discussions happens